kbase-extension/static/kbase/js/common/jobManager.js

```diff
@@ -155,7 +155,8 @@ define(['common/dialogMessages', 'common/jobs', 'common/jobCommChannel'], (
          *  {string}    handlerName - the name of the handler to add
          *  {function}  handlerFn - the function to be performed
          *              if the handler function is already installed on the job manager,
-         *              'null' can be supplied in place of the function
+         *              `handlerFn` will replace the existing function. If the existing
+         *              function can be used, `null` can be supplied.
```

Comment:
change of function here -- replace existing handlers instead of leaving them be.